Majasdarbs/04-majasdarbs/cv-css/css/style.css

```diff
@@ -0,0 +1,159 @@
+@import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap');
+section {
+    padding: 50px;
+    min-height: 10vh;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    max-width: 900px;
+    margin: auto;
+  }
+  hr{
+    max-width: 900px;
+  }
+  address{
+    max-width: 900px;
+  }
+  body {
+    background-color: #252422;
+    color: rgb(189, 189, 189);
+    font-family: Consolas,monaco,monospace;
+    margin: auto;
+  }
+  h1 {
+    font-weight: 700;
+    font-size: 2rem;
```

Comment:
you are using different units for font-sizes. Better use rems everywhere (not just here)